third_party/blink/renderer/modules/cobalt/crash_annotator/crash_annotator.idl

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Allows (key, value) pairs to be added to uploaded crash reports as context.
+[
+    Exposed=Window,
+    SecureContext
+]
+interface CrashAnnotator {
```

Comment:
We'll have to discuss more with Kabuki about whether they want to keep the H5vcc prefix or not—some initial checks with them make it seem like they do, but that's probably a good thing to concretely bring up in your H5vcc survey doc. I'll leave it to your discretion for now

Reply:
Thanks, I think that's a good idea to discuss with the Kabuki team as part of the h5vcc survey. I'll leave the naming as is in the PR for now but adjust later based on the outcome of the h5vcc survey review.